api/ServiceWorkerContainer.json

```diff
@@ -406,14 +406,13 @@
           "spec_url": "https://w3c.github.io/ServiceWorker/#dom-serviceworkerglobalscope-onmessageerror",
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": "80"
             },
             "chrome_android": {
-              "version_added": false
+              "version_added": "80"
             },
             "edge": {
-              "version_added": "17",
-              "version_removed": "79"
```

Comment:
Was it really in 79, or actually missing in just that one version as the Chrome data would suggest?

Reply:
I've run https://mdn-bcd-collector.appspot.com/tests/api/ServiceWorkerContainer/onmessageerror in Edge 16, 17, 18, 79 and 80 to confirm. It was added in 17, not in 79, and then in 80 again as one would expect. I'll update the data here.